<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WIN-AEE4QQFEOGF" name="com.youceedu.interf.test.TestRun" tests="2" failures="1" timestamp="6 Jun 2021 13:32:57 GMT" time="2.434" errors="0">
  <testcase name="httpReq" time="0.176" classname="com.youceedu.interf.test.TestRun">
    <failure type="com.alibaba.fastjson.JSONException" message="syntax error, pos 1, line 1, column 2您的服务器返回码为404请检查您的地址">
      <![CDATA[com.alibaba.fastjson.JSONException: syntax error, pos 1, line 1, column 2您的服务器返回码为404请检查您的地址
	at com.alibaba.fastjson.parser.DefaultJSONParser.parse(DefaultJSONParser.java:1497)
	at com.alibaba.fastjson.parser.DefaultJSONParser.parse(DefaultJSONParser.java:1383)
	at com.alibaba.fastjson.JSON.parse(JSON.java:182)
	at com.alibaba.fastjson.JSON.parse(JSON.java:192)
	at com.alibaba.fastjson.JSON.parse(JSON.java:148)
	at com.alibaba.fastjson.JSONPath.read(JSONPath.java:688)
	at com.youceedu.interf.util.PatternUtil.conpareResultTodb(PatternUtil.java:79)
	at com.youceedu.interf.test.TestRun.httpReq(TestRun.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- httpReq -->
  <testcase name="httpReq" time="2.258" classname="com.youceedu.interf.test.TestRun"/>
</testsuite> <!-- com.youceedu.interf.test.TestRun -->
